ethcore/sync/Cargo.toml

```diff
@@ -9,6 +9,7 @@ edition = "2018"
 [lib]
 
 [dependencies]
+elapsed = "0.1.2"
```

Comment:
https://github.com/matklad/elapsed/commit/4f2c81af68d29ebca19fd7ca2fb1b4f312436d02
this library is deprecated, we should just use std::time